src/ResourceManager/Sql/Commands.Sql/SqlDatabaseAgent/Cmdlet/JobCredential/GetAzureSqlDatabaseAgentJobCredential.cs

```diff
@@ -0,0 +1,135 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Sql.SqlDatabaseAgent.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Rest.Azure;
+
+namespace Microsoft.Azure.Commands.Sql.SqlDatabaseAgent.Cmdlet
+{
+    /// <summary>
+    /// Defines the Get-AzureRmSqlDatabaseAgentJobCredential Cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseAgentJobCredential", 
+        SupportsShouldProcess = true,
```

Comment:
Get cmdlets do not need SupportShouldProcess - please remove from all Get cmdlets.